man/sd_bus_add_object.xml

```diff
@@ -508,7 +508,7 @@
         <varlistentry>
           <term><constant>SD_BUS_VTABLE_METHOD_NO_REPLY</constant></term>
 
-          <listitem><para>Mark his vtable entry as a method that will not return a reply using the
```

Comment:
this is technically a typo fix

Reply:
it's ok, we often update/reformat many docs in the same commit when doing fixups like these